mirascope/core/base/_utils/_convert_function_to_base_tool.py

```diff
@@ -116,7 +116,7 @@ def call(self: base):
             )
         )
 
-    async def call_async(self: base):
+    async def call_async(self: base) -> Awaitable[Callable]:
```

Comment:
again, wouldn't the return type here just be `Callable` since it's marked `async`?